lib/omnibus/config.rb

```diff
@@ -400,14 +413,22 @@ def reset!
     #
     # @return [String]
     default(:publish_s3_access_key) do
```

Comment:
Nevermind. I just realized the difference between `populate` and `publish`; where `populate` uploads build dependencies and `publish` is the resulting build artifact(s). Nevertheless, I probably should either remove the changes to `publish_s3_*`or increase the scope of the change to include the publisher.